generator/generator_tests/mini_roundabout_tests.cpp

```diff
@@ -0,0 +1,182 @@
+#include "testing/testing.hpp"
+
+#include "coding/point_coding.hpp"
+
+#include "base/math.hpp"
+#include "geometry/latlon.hpp"
+#include "geometry/point2d.hpp"
+#include "geometry/mercator.hpp"
+#include "generator/osm_element.hpp"
+#include "generator/mini_roundabout_processor.hpp"
+
+#include <algorithm>
+#include <vector>
+#include <math.h>
```

Comment:
`#include <cmath>`?

Reply:
Removed this include from tests. It is not used anymore. Changed `math.h` to `cmath`
 in `mini_roundabout_processor.cpp`.